lib/prediction_class.py

```diff
@@ -370,6 +387,13 @@ class MHCII(PredictionClass, metaclass=ABCMeta):
     def needs_epitope_length(self):
         return False
 
+    @classmethod
+    def species_for_allele(self, allele):
+        if allele.startswith('H2'):
+            return 'mouse'
+        else:
+            return 'human'
+
```

Comment:
Is the difference between this and the above that no Class II algorithms support anything but human or mouse?  Would it be better to make a comprehensive list of class I and II prefixes in a dictionary (based on the valid_alleles list) and then we'd only have one function to update as new tools come online?

Reply:
For class I mouse alleles start with `H-2` and for class II they start with `H2`. We can still combined them, I think.

Reply:
I just pushed a commit to consolidate these two methods.